### DISCUSSION TITLE
Won't Read ASCII charaacters properly/Converting

### DISCUSSION THREAD

**Discussion by Vanir-Labs** [ImageMagick/ImageMagick#6601] [Help] [answered]
### ImageMagick version

7.1.1.15 Q16-HDRI 64x

### Operating system, version and so on

Windows 10 64x

### Description

When I enter most ACSII characters it processes normally, but there are a handful of ones that seem to not read properly.
Here is an example:

My folder "CE¥ìa©öU¢¥U" is read as "CEï¿½ï¿½acï¿½Uï¿½ï¿½U" in the CMD window during execution and does no process these files in the proper locations because the name has been changed.
Anyone know how to make it read all ACSII characters properly? Maybe it's a CMD issue instead?

**Comment by snibgo** [comment]
"CE¥ìa©öU¢¥U" are not ASCII characters. They are characters that might be encoded as UTF-8, or some Windows code page, or something else.

What is your current codepage? The Windows command `chcp` will tell you. Windows codepage 65001 may be the one you need, to process characters encoded as UTF-8.

Windows 11 seems to handle UTF-8 better than Windows 8.1. I have never used Windows 10.

**Reply by Vanir-Labs** [reply]
My code page starts with this
```
chcp 65001 /u
convert "C:\Users\rivers\Desktop\data\texture\CE¥ìa©öU¢¥U\01.bmp" -type truecolor "C:\Users\rivers\Desktop\data\texture\CE¥ìa©öU¢¥U\01.bmp"
```
EDIT: I also used mogrify instead of convert.

**Comment by Vanir-Labs** [comment, marked as answer]
![image](https://github.com/ImageMagick/ImageMagick/assets/82656075/72681b1a-1a86-465e-b7e9-675b74de2987)
Thanks for the help. I ended up downloading a japanese console and it solved all my problems.

**Comment by Vanir-Labs** [comment]
Language barrier on Console. Download a new Console language pack or new language specific console.
